sources/packages/backend/libs/services/src/constants/system-configurations-constants.ts

```diff
@@ -6,3 +6,13 @@ import { Duration } from "zeebe-node";
  */
 export const ZEEBE_PUBLISH_MESSAGE_DEFAULT_TIME_TO_LEAVE =
   Duration.seconds.of(10);
+
+/**
+ * Number of attempts that a queue will re-try a failed job.
+ */
+export const QUEUE_RETRY_ATTEMPTS = 3;
```

Comment:
Maybe those variables can be renamed to include `default` like `QUEUE_DEFAUL_RETRY_ATTEMPTS`.
Whatever we do not know what to use we go for these ones.
Also, it can be the individual properties or the object as below:
```ts
export const QUEUE_RETRY_DEFAULT_CONFIG = { attempts: QUEUE_RETRY_ATTEMPTS, backoff: QUEUE_RETRY_DELAY };
```